### ISSUE TITLE
The train speed of classification network

### ISSUE THREAD

**Issue by TSKongLingwei** [jiwoon-ahn/psa#20]
Hello, When I train the classification Network, i find it is too slow. It almost take 1 second for each step with batch-size 4. Data loader is the most Time-consuming part. It is also very slow when calculate the loss with x and label. The forward part and the background part are quick. I test it on pytorch0.4.0 and pytorch 1.0. The results are similar. Is this normal? Thank you very much.
